src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6591,4 +6591,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_UnreadRecordParameter_Title" xml:space="preserve">
     <value>Parameter is unread. Did you forget to use it to initialize the property with that name?</value>
   </data>
+  <data name="WRN_DoNotCompareFunctionPointers" xml:space="preserve">
+    <value>Function pointer values should not be compared, as JIT tiering and recompilation means that taking the address of a method can return different addresses at different times.</value>
```

Comment:
>, as JIT tiering and recompilation means that taking the address of a method can return different addresses at different times [](start = 57, length = 126)

The explanation seems unnecessarily specific. How about "since pointers to the same function may have distinct addresses at runtime."